File: app/src/main/java/com/shakil/barivara/utils/SpinnerAdapter.java
package com.shakil.barivara.utils;

import android.content.Context;
import android.widget.ArrayAdapter;
import android.widget.Spinner;

import com.shakil.barivara.R;

import java.util.ArrayList;

public class SpinnerAdapter {

    public void setSpinnerAdapter(Spinner spinner, Context context, ArrayList<String> spinnerValue){
        ArrayAdapter<String> spinnerAdapter = new ArrayAdapter<>(context, R.layout.spinner_drop, spinnerValue);
        spinnerAdapter.setDropDownViewResource(R.layout.simple_spinner_dropdown_item);
        spinner.setAdapter(spinnerAdapter);
    }
}

File: app/src/main/java/com/shakil/barivara/utils/SpinnerAdapter.kt
package com.shakil.barivara.utils

import android.content.Context
import android.widget.ArrayAdapter
import android.widget.Spinner
import com.shakil.barivara.R

class SpinnerAdapter {
    fun setSpinnerAdapter(spinner: Spinner, context: Context?, spinnerValue: ArrayList<String>) {
        val spinnerAdapter = ArrayAdapter(context!!, R.layout.spinner_drop, spinnerValue)
        spinnerAdapter.setDropDownViewResource(R.layout.simple_spinner_dropdown_item)
        spinner.adapter = spinnerAdapter
    }
}
